src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends AbstractFigure {
+    private String color;
```

Comment:
The `color` field is redundant here because it is already defined in the `AbstractFigure` class. You should remove this field from the `Square` class to avoid confusion and potential errors.